libs/component-library/src/lib/button-group/buttonGroup.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { FxBox } from '../box/box';
+import { FxPressableOpacity } from '../pressable-opacity/pressableOpacity';
+import { FxText } from '../text/text';
+
+type BgButtonProps = {
+  text: string;
+  selected: boolean;
+  onPress: () => void;
+  disabled: boolean;
+};
+
+const BgButton = ({ text, selected, onPress, disabled }: BgButtonProps) => {
+  return (
+    <FxPressableOpacity
+      flex={1}
+      alignItems="center"
+      justifyContent="center"
+      height={40}
+      backgroundColor={
+        selected ? (disabled ? 'backgroundSecondary' : 'content3') : undefined
+      }
+      onPress={onPress}
+      disabled={selected || disabled}
+      hitSlop={0}
+    >
+      <FxText
+        color={
+          selected
+            ? disabled
+              ? 'border'
+              : 'backgroundApp'
+            : disabled
+            ? 'backgroundSecondary'
+            : 'content3'
+        }
+      >
+        {text}
+      </FxText>
+    </FxPressableOpacity>
+  );
+};
+
+type FxButtonGroupProps = {
+  items: Array<string>;
+  selectedIdx?: number | null;
+  onSelect: (idx: number) => void;
+  disabled?: boolean;
+};
+export const FxButtonGroup = ({
+  items,
+  selectedIdx = null,
+  disabled = false,
+  onSelect,
+}: FxButtonGroupProps) => {
+  const selectHandler = (idx: number) => {
+    if (idx !== selectedIdx) {
+      onSelect(idx);
+    }
+  };
+
+  return (
+    <FxBox
+      flexDirection="row"
+      borderWidth={1}
+      borderColor={disabled ? 'backgroundSecondary' : 'content3'}
+      borderRadius="s"
```

Comment:
Could pass `...rest' to FxBox where `...rest` is FxBox props, allowing user of comp to pass margin etc to FxButton instead of having to wrap FxButton in another FxBox to hand layout

Reply:
done